### ISSUE TITLE
Amazing result

### ISSUE THREAD

**Issue by zhanziwei** [DeepAlchemist/deep-person-search#8]
Hi,  I tested your pretrained model, and I got some amazing result.My test script was `python main.py --is_test --benchmark prw --eval_batch_size 5  --backbone bsl --in_level C5 --cls_type oim --load_ckpt /workplace/.torch/checkpoints/prw-bsl.pth`. And I got this. 

![Xshot-0014.png](https://i.loli.net/2020/11/24/H8G3v5ZIyeSPwCb.png)
Rank-1 is the top-1 in the picture? thanks.

**Comment by deepalchemist** [comment]
WoW, It seems that the detection performance is normal, but the ReID performance is abnormal.
You should make sure that you have not modified any evaluation codes and datasets.

**Comment by YangJae96** [comment]
Hi, I also tested your code and got same results with above question.
python main.py --is_test --benchmark prw --eval_batch_size 5 --backbone bsl --in_level C5 --cls_type oim

![image](https://user-images.githubusercontent.com/57695621/106462047-eacdc700-64d8-11eb-9040-9c9c0b351ab3.png)

But it does not match the result with your table.
I did not change the eval code. 

Is there something wrong??
Thanks in advance.
